package.json

```diff
@@ -47,17 +47,17 @@
   },
   "devDependencies": {
     "@semantic-release/changelog": "^6.0.3",
-    "@semantic-release/commit-analyzer": "^11.1.0",
+    "@semantic-release/commit-analyzer": "^10.0.4",
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/github": "^9.2.6",
-    "@semantic-release/npm": "^11.0.2",
-    "@semantic-release/release-notes-generator": "^12.1.0",
+    "@semantic-release/npm": "^10.0.3",
+    "@semantic-release/release-notes-generator": "^11.0.7",
     "@types/long": "^5.0.0",
-    "@types/node": "^20.11.20",
-    "conventional-changelog-conventionalcommits": "^7.0.2",
+    "@types/node": "^20.16.2",
+    "conventional-changelog-conventionalcommits": "^8.0.0",
     "loose-ts-check": "^2.0.0",
-    "semantic-release": "^23.0.2",
-    "typescript": "^5.3.3"
+    "semantic-release": "^21.1.1",
+    "typescript": "^5.5.4"
```

Comment:
See if this supports node v18 or what last version does

Reply:
It does